python/main.py

```diff
@@ -91,23 +94,32 @@ def root():
 @app.post("/items")
 async def add_one_item(name: str = Form(...),
                        category: str = Form(...),
-                       image: str = Form(...)):
+                       image_filename: UploadFile = File(...)):
+
+    #save the uploaded file
+    filename = image_filename.filename
+    hashed_filename = hash_image(filename)
+    save_path = images / hashed_filename
+
+    with open(save_path, 'wb') as buffer:
+        shutil.copyfileobj(image_filename.file, buffer)
```

Comment:
there's a possibility saving will cause error

Reply:
instead of multiple try ... except, maybe it's better to put one error handling at the bottom?

Reply:
Thank you very much！I will add the error handling for it

Reply:
> instead of multiple try ... except, maybe it's better to put one error handling at the bottom?

Regarding "one error handling", is it fine to mix error handling of saving-image part and insert-data-into-database  part together by using only one try .. except? 😳